modules/gdscript/SCsub

```diff
@@ -17,3 +17,7 @@ if env["tools"]:
         # Using a define in the disabled case, to avoid having an extra define
         # in regular builds where all modules are enabled.
         env_gdscript.Append(CPPDEFINES=["GDSCRIPT_NO_LSP"])
+
+if env["tests"]:
+    env_gdscript.Append(CPPDEFINES=["TESTS_ENABLED"])
```

Comment:
Is this necessary? It should already be inherited from the main `env` which was cloned for `env_gdscript` no?

Reply:
Yes, because we're actually outside of the `main/` env here. Recall #40726, this is what I meant when saying that some modules may eventually need this. Defining `TESTS_ENABLED` for all modules is also an option, but GDScript may be just an exception.